README.md

```diff
@@ -154,6 +152,33 @@ YACHT can be run via the command line `yacht <module>`. Now it has three four ma
   | k                 | the length of k-mer |
   | outfolder         | the path to a folder where the downloaded file is expected to locate |
 
+- The `sketch` module has two submodules: `ref` and `sample`:
```

Comment:
Somewhere we should say that this is a simple wrapper to `sourmash`

Reply:
I have added a note to emphasize that it is a simple wrapper to `sourmash`, See [here](https://github.com/KoslickiLab/YACHT/blob/474208ff6845d9a634737e180cb4681773af0397/README.md?plain=1#L155).